dcs/triggers.py

```diff
@@ -11,6 +11,11 @@
 if TYPE_CHECKING:
     from dcs.terrain import Terrain
 
+class TriggerColor(Enum):
+    Red = "0xff0000ff"
```

Comment:
Maybe add the `White` also here?
I guess the rest of the colors are the same as in DCS predefined?